packages/design-system/src/components/MultiSelectField/style.scss

```diff
@@ -57,9 +57,8 @@ $base-class: 'multiselect';
   &-head__search {
     display: block;
     width: 5px;
-    margin: 2px 3px;
-    font-size: 16px;
-    line-height: 28px;
+    margin: 5px 10px;
+    line-height: 1.3;
```

Comment:
I believe that ie11 or safari will give you a hard time after this change

Reply:
you mean line height?

Reply:
let's just test it, maybe it will work just fine xD